### ISSUE TITLE
scipy dependency? 

### ISSUE THREAD

**Issue by luzpaz** [looooo/freecad.frametools#10]
Is there one? and what is the minimum version?  
Note: add dep tag to `package.xml` if necessary

**Comment by looooo** [comment]
https://github.com/looooo/freecad.frametools/commit/af8f4b0c506ba097678de1b68dcf60f8a2be0e9c

but does it check if there is already a SciPy installed? I don't want to install SciPy if it is already installed in a bundle via another package-manager

**Comment by luzpaz** [comment]
@chennes does the addon manager check in advance if scipy is installed?

@looooo what's the minimum version required?

**Comment by chennes** [comment]
Yes, the Addon Manager tries to import whatever library you specify, and if that import works, it doesn't try to install it.
